packages/@aws-cdk/aws-glue/README.md

```diff
@@ -23,6 +23,55 @@
 
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+## Connection
+
+A `Connection` allows Glue jobs, crawlers and development endpoints to access certain types of data stores.
+
+```ts
+new glue.Connection(stack, 'MyConnection', {
+  connectionType: glue.ConnectionTypes.NETWORK,
+  securityGroups: [securityGroup],
+  subnet,
+});
+```
+
+If you need to use a connection type that doesn't exist as a static member on `ConnectionType`, you can instantiate a `ConnectionType` object, e.g: `new glue.ConnectionType('NEW_TYPE')`.
+
+Different connection types require different connection properties which can be set via `ConnectionProps.properties`. 
```

Comment:
No need.

Reply:
Do you mean line 40 or 38 as well? I think for 38 you mentioned it should be here - from the doc class docs?

Reply:
Dropped L40